File: app/src/main/java/cn/onlinecache/breakpad/BreakpadManager.java
package cn.onlinecache.breakpad;

import java.io.File;

public class BreakpadManager {
	private static boolean mWasSetup;

	
	public static void setup(File file) {
		if (mWasSetup) {
			return;
		}
		
		try {
			NativeBreakpad.init(file.getAbsolutePath());
			mWasSetup = true;
		} catch(Throwable e) {

		}
	}
	
	public static void testNativeCrash() {
		NativeBreakpad.testNativeCrash();
	}
}

File: app/src/main/java/cn/onlinecache/breakpad/BreakpadManager.kt
package cn.onlinecache.breakpad

import java.io.File

object BreakpadManager {

    private var wasSetup = false

    fun setup(file: File) {
        if (wasSetup) {
            return
        }
        runCatching {
            NativeBreakpad.init(file.absolutePath)
            wasSetup = true
        }
    }

    fun testNativeCrash() {
        NativeBreakpad.testNativeCrash()
    }
}